### PR TITLE
[REFACTOR] some small name changes

### PR DESCRIPTION
**Pull request by timosachsenberg**
# Description

Minor name changes to better reflect design.

# Checklist:
- [X] Make sure that you are listed in the AUTHORS file
- [X] Add relevant changes and new features to the CHANGELOG file
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [X] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

### CODE REVIEW

File: src/openms_gui/include/OpenMS/VISUAL/sources.cmake [hunk 1]
```diff
@@ -50,11 +50,11 @@ TOPPASToolVertex.h
 TOPPASTreeView.h
 TOPPASVertex.h
 TOPPASWidget.h
-TOPPViewIdentificationViewBehavior.h
 TOPPViewMenu.h
 #TOPPViewPreferences.h
```
Comment (oliveralka) [inline src/openms_gui/include/OpenMS/VISUAL/sources.cmake:54]: can that be removed?

File: src/openms_gui/source/VISUAL/DataSelectionTabs.cpp [hunk 2]
```diff
@@ -61,24 +61,24 @@ namespace OpenMS
     : QTabWidget(parent),
     spectra_view_widget_(new SpectraViewWidget(this)),
     id_view_widget_(new SpectraIdentificationViewWidget(Param(), this)),
-    spectraview_behavior_(new TOPPViewSpectraViewBehavior(tv)),
-    idview_behaviour_(new TOPPViewIdentificationViewBehavior(tv, id_view_widget_)),
+    spectraview_behavior_(new TVSpectraViewController(tv)),
```
Comment (oliveralka) [inline src/openms_gui/source/VISUAL/DataSelectionTabs.cpp:64]: maybe rename variable to 
`spectraview_controller_`
`idview_controller_`
for consistency?

File: src/openms_gui/source/VISUAL/sources.cmake [hunk 3]
```diff
@@ -56,11 +56,11 @@ TOPPASToolVertex.cpp
 TOPPASTreeView.cpp
 TOPPASVertex.cpp
 TOPPASWidget.cpp
-TOPPViewIdentificationViewBehavior.cpp
 TOPPViewMenu.cpp
 #TOPPViewPreferences.cpp
```
Comment (oliveralka) [inline src/openms_gui/source/VISUAL/sources.cmake:60]: maybe remove the unused entry.

**Comment by timosachsenberg** [comment]
thanks will update and notify for review

File: src/openms_gui/include/OpenMS/VISUAL/sources.cmake (hunk 1 above)
Comment (timosachsenberg) [reply src/openms_gui/include/OpenMS/VISUAL/sources.cmake:54]: yes was renamed to TVIdentificationViewController

File: src/openms_gui/source/VISUAL/sources.cmake (hunk 3 above)
Comment (timosachsenberg) [reply src/openms_gui/source/VISUAL/sources.cmake:60]: its from chris... not sure about it

**Comment by timosachsenberg** [comment]
THere seems to be an unrelated test failing.

**Comment by oliveralka** [comment]
rebuild jenkins

**Review by oliveralka — APPROVED** [review]
looks good!

**Review by cbielow — APPROVED** [review]

**Pull request merged**